helm-charts/konk-service/values.yaml

```diff
@@ -50,6 +50,18 @@ service:
 crds: null
 
 group:
+  name: example.com
```

Comment:
It looks like this is relevant only to the `example-apiserver` chart, can we add this directly to the `KonkService` object in that chart?

Reply:
I changed the parameters. Group is now an object with properties so we can configure rbac permissions